WireUI/Sources/WireConversationUI/ConversationCell/CellTypes/ConversationCellModel+refreshInterval.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+public extension ConversationCellModel {
+
+    var refreshInterval: TimeInterval {
+        switch self {
+        case .timeDivider(let timeDividerModel):
+            10
+        default:
+            .zero
```

Comment:
Usually cells shouldn't need to be updated periodically.
But it will make sense for time dividers and self-deleting message cells, if they contain a countdown label.